### ISSUE TITLE
Create Profile Cards with Hover Effects

### ISSUE THREAD

**Issue by gautamankoji** [mosaicra-ui/mosaicra-ui#8]
**Description:**
Design profile cards that display user information, including hover interactions and responsive design.

**Skills Required:**
HTML, CSS, JavaScript.

**Comment by uzefaqureshi06** [comment]
can you assign this to me i want to work on it

**Comment by uzefaqureshi06** [comment]
can you plz assign this to me I want to work on this

**Comment by uzefaqureshi06** [comment]
can you plz assign this to me i want ot work on it

**Comment by uzefaqureshi06** [comment]
@gautamankoji  hey I had send you a PR can you please review it and merge it

**Comment by gautamankoji** [comment]
@uzefaqureshi06 
yes your PR #22 has been accepted and an review is been added too make sure to check that out!
